toolkit/geoview-compose/src/main/java/com/arcgismaps/toolkit/geoviewcompose/GeoViewScope.kt

```diff
@@ -175,6 +180,10 @@ public sealed class GeoViewScope protected constructor(private val geoView: GeoV
             allowCalloutRecomposition = true
             this.CalloutInternal(geoElement, modifier, tapLocation, colorScheme, shapes, content)
         }
+
+        SideEffect {
+            reset()
+        }
```

Comment:
Suggest to move this up into the if block above.

```suggestion

        SideEffect {
            reset()
        }
```